plugins/net_plugin/net_plugin.cpp

```diff
@@ -439,9 +439,9 @@ namespace eosio {
 
    public:
       void update_chain_info();
-      void update_chain_info(const block_id_type& lib);
+      void update_chain_info(const block_id_type& froot_id);
       chain_info_t get_chain_info() const;
-      uint32_t get_chain_lib_num() const;
+      uint32_t get_fork_root_num() const;
```

Comment:
`froot` here to I think for consistency.

Reply:
Hmm, we have a `get_fork_head_num`. Seems better to spell it out.